### ISSUE TITLE
Arm architecture not supported

### ISSUE THREAD

**Issue by samip5** [nginx-proxy/nginx-proxy#1370]
### What happened?
I tried to execute this container on armv7 platform, which unfortunately fails.

### What you expected to happen?
I expected it to work out-of-the box.

### Error log
```
standard_init_linux.go:211: exec user process caused "exec format error"
```

`docker image inspect jwilder/nginx-proxy |grep -i "Arch"`:
```
"Architecture": "amd64"
```

Related: #1262.

**Comment by danywheeler** [comment]
what's the status? I use arm dev board do some test, but couldn't find nginx-proxy arm64 version!

**Comment by samip5** [comment]
> what's the status? I use arm dev board do some test, but couldn't find nginx-proxy arm64 version!

There's no progress, but they're multiple nginx containers that have the support, like [linuxserver/nginx](https://hub.docker.com/r/linuxserver/nginx).

**Comment by utrumo** [comment]
Hi!
I have some problem with arm processor on Zyxel Armor Z2:
`standard_init_linux.go:211: exec user process caused "exec format error"`

As an experiment i cloned this repository and changed references in Dockerfile:
- changed link to forego on ARM version 20180217041714 (i got it from https://github.com/ddollar/forego)
```
# Install Forego
RUN wget https://bin.equinox.io/c/ekMN3bCZFUn/forego-stable-linux-arm.tgz \
 && tar xvf forego-stable-linux-arm.tgz -C /usr/local/bin \
 && rm forego-stable-linux-arm.tgz
```

- changed docker-gen to armhf version:
```
RUN wget https://github.com/jwilder/docker-gen/releases/download/$DOCKER_GEN_VERSION/docker-gen-linux-armhf-$DOCKER_GEN_VERSION.tar.gz \
 && tar -C /usr/local/bin -xvzf docker-gen-linux-armhf-$DOCKER_GEN_VERSION.tar.gz \
 && rm docker-gen-linux-armhf-$DOCKER_GEN_VERSION.tar.gz
```

- then just built it:
```
docker build -t nginx-proxy:arm .
```
- and changed in my docker-conmpose.yml image from
```
  nginx-proxy:
    image: jwilder/nginx-proxy
```
to
```
  nginx-proxy:
    image: nginx-proxy:arm
```
And it starts working.

**Comment by utrumo** [comment]
I made a little more concise decision:
Made a  [nginx-gen](https://hub.docker.com/repository/docker/utrumo/nginx-gen/) image with docker-gen that runs on amd64 and arm/v7

Here 's an example of how it can be used: [docker-compose.yml](https://github.com/utrumo/docker-nginx-proxy-with-letsencrypt/blob/master/docker-compose.yml)

**Comment by MattJeanes** [comment]
I've made an unofficial ARM64 build here in the meantime, figure it may be useful to others: https://hub.docker.com/r/mattjeanes/jwilder-nginx-proxy-arm64

**Comment by jelle619** [comment]
Thank you so much! @MattJeanes

**Comment by rt87** [comment]
I know I can compile the image for arm architecture myself... but that is another step that I have to perform. Hence, I still would appreciate a cross-architecture compile which includes the arm architecture.

**Comment by buchdag** [comment]
Fixed as of #1470 + 4a1d5bfc58af35f5e90dcefb47b35a274c903150 + d120ce9fdebf09a6c7b90ecff7eda0b7d2760ef3

`arm64` and `arm/v7` images are availables from `jwilder/nginx-proxy` or the new registry `nginxproxy/nginx-proxy`
